test/testcafe/framework/page-objects/PlaygroundErrorPageObject.js

```diff
@@ -9,6 +9,6 @@ export default class PlaygroundErrorPageObject extends BasePageObject {
   }
 
   hasTitle() {
-    return !!this.title;
```

Comment:
this was causing the function to always return true, making the spec test always pass.  The spec test has not been working properly until now due to this.  Thats probably why we didn't catch this missing view earlier

Reply:
Thanks for fixing! I was going to write a comment about the return type of `this.title.exists` being `Promise<boolean>` and that you'd need to `await` it but TIL that testcafe auto `await`s certain Promises: https://testcafe.io/documentation/402702/reference/test-api/testcontroller/expect#:~:text=TestCafe%20awaits%20Promises%20from%20compatible%20asynchronous%20functions

Reply:
ohh yeah good point.   Interesting okay cool good to know!